ui/hooks/gasFeeInput/useMaxFeePerGasInput.js

```diff
@@ -53,24 +60,31 @@ export function useMaxFeePerGasInput({
 
   const showFiat = useSelector(getShouldShowFiat);
 
-  const maxFeePerGasFromTransaction = supportsEIP1559
-    ? getMaxFeePerGasFromTransaction(transaction)
-    : 0;
+  let initialMaxFeePerGas;
+  if (editGasMode === EDIT_GAS_MODES.SWAPS && supportsEIP1559V2) {
+    initialMaxFeePerGas = swapCustomMaxFeePerGas
+      ? Number(hexWEIToDecGWEI(swapCustomMaxFeePerGas))
+      : null;
```

Comment:
Should this be `0` instead of null?

Reply:
This is to ensure that null coalescing operator used below works well.

Reply:
Could you be a bit more specific? Not following which instance of null coalescing your talking about here? on new line 128? If so, that is using `maxFeePerGas` not `initalMaxFeePerGas`?